tests/formattedcode/data/spdx/license_known/expected.tv

```diff
@@ -1,35 +1,36 @@
-# Document Information
+## Document Information
```

Comment:
Why using a double ## pound?

Reply:
Regarding this and similar questions about the change of output:
During the refactoring of the python-tools we had to rewrite most of the serializers. For this, we followed the practices employed by the [official SPDX spec examples](https://github.com/spdx/spdx-spec/blob/development/v2.3.1/examples/SPDXTagExample-v2.3.spdx), especially the ordering of information and formatting of comments (which use double ## and "Information" in most places).

Reply:
It does not make any sense: ## makes the file bigger for no good reason. The fact that spec examples use this weird way does not make it part of the spec FWIW. The spec states that a # is a comment not that you should use a double ##. Now this is not critical so I can live with this change but this feels like cargo culting.

But the logical order from most significant to least significant is namespace then name. This is BTW also the order of the spec examples you reference https://github.com/spdx/spdx-spec/blob/development/v2.3.1/examples/SPDXTagExample-v2.3.spdx#L3 

I do not care too much for the change so I can live with it  but there is no raionale in this change that I can understand.